types/underscore/index.d.ts

```diff
@@ -5704,7 +5704,7 @@ declare module _ {
         * Wrapped type `object`.
         * @see _.pairs
         **/
-        pairs(): _Chain<T[]>;
+        pairs(): _Chain<any[]>;
```

Comment:
Why the weakening from `T` to `any` here? Was `T` the wrong type?
Is this related to the change to `pairs` [above](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27804/files#diff-bc995fed3367d28165753a2581d1af54R3634)? If that's the case, would this not be `[string, any][]`?